src/Http/Controllers/Views/Projects/LargoController.php

```diff
@@ -36,11 +38,32 @@ public function index(Request $request, $id)
         $patchUrlTemplate = Storage::disk(config('largo.patch_storage_disk'))
             ->url(':prefix/:id.'.config('largo.patch_format'));
 
+        $shapes = collect([
+            1 => 'Point',
+            2 => 'LineString',
+            3 => 'Polygon',
+            4 => 'Circle',
+            5 => 'Rectangle',
+            6 => 'Ellipse',
+            7 => 'WholeFrame',
+        ]);
+
+        $usersWithAnnotations = ImageAnnotationLabel::query()
+            ->join('image_annotations', 'image_annotations.id', '=', 'image_annotation_labels.annotation_id')
+            ->join('images', 'image_annotations.image_id', '=', 'images.id')
+            ->where('images.volume_id', $id)
+            ->join('users', 'image_annotation_labels.user_id', '=', 'users.id')
+            ->distinct('image_annotation_labels.user_id')
+            ->select('image_annotation_labels.user_id', 'users.lastname', 'users.firstname')
+            ->get();
```

Comment:
I'd prefer to keep this as an extra API call. If you inject it on every page, the data is always there although not all users will use the filter by user feature. As this is a rather expensive query, we should avoid it unless we need it.